# Python if else within a try and except
except Exception as e:
    print("Error has occurred", e)
